sci-electronics/verilator/verilator-3.924.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A fast C++ Verilog simulator"
+DESCRIPTION="Verilator is the fastest free Verilog HDL simulator, and \
+beats most commercial simulators. It compiles synthesizable Verilog (not \
+test-bench code!), plus some PSL, SystemVerilog and Synthesis assertions \
+into C++ or SystemC code. It is designed for large projects where fast \
+simulation performance is of primary concern, and is especially well suited \
+to generate executable models of CPUs for embedded software design teams."
+HOMEPAGE="https://www.veripool.org/"
+COPYRIGHT="2006-2018 Wilson Snyder"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://www.veripool.org/ftp/verilator-$portVersion.tgz"
+CHECKSUM_SHA256="7dcb19711b8630ada59f0d3d7409faa9649e37bf4c53a0bbfcad32afb28b5975"
+PATCHES="verilator-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+ 	commandSuffix=
+ 	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	verilator$secondaryArchSuffix = $portVersion
+	cmd:verilator$commandSuffix = $portVersion
+	cmd:verilator_bin$commandSuffix = $portVersion
+	cmd:verilator_bin_dbg$commandSuffix = $portVersion
+	cmd:verilator_coverage$commandSuffix = $portVersion
+	cmd:verilator_coverage_bin_dbg$commandSuffix = $portVersion
+	cmd:verilator_profcfunc$commandSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:bison
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:perl
+	"
+
+BUILD()
+{
+	runConfigure --omit-dirs "binDir" ./configure \
+		--bindir="$commandBinDir"
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	mkdir -p $developLibDir/pkgconfig $includeDir
+	mv $dataDir/pkgconfig/* $developLibDir/pkgconfig/
```

Comment:
I think they have to be in `libDir` for `fixPkgconfig` to work. But I might be wrong. Please check the resulting `.pc` files.

Reply:
removed fixPkgconfig